### ISSUE TITLE
How to change the user id column?

### ISSUE THREAD

**Issue by crabnky** [msgphp/symfony-demo-app#20]
Hi again,

Sorry for being so importunate, and may be asking for trivial things. :|
And thank you for your patience.
I came across another issue - I have to rename the `id` column of `users` table to `id_users` (this is a requirement from our database guy) and I wonder how I can do this.

Here is my users table mapping:
```
<?xml version="1.0" encoding="utf-8"?>
<doctrine-mapping xmlns="http://doctrine-project.org/schemas/orm/doctrine-mapping"
                  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
                  xsi:schemaLocation="http://doctrine-project.org/schemas/orm/doctrine-mapping http://doctrine-project.org/schemas/orm/doctrine-mapping.xsd">

    <entity name="App\Entity\User" table="tbl_users" repository-class="App\Repository\UserRepository">

        <field name="firstName" column="first_name" type="string" length="50" nullable="true"/>
        <field name="lastName" column="last_name" type="string" length="50" nullable="true"/>
        <field name="abbreviation" column="abbreviation" type="string" length="3" nullable="true"/>
        <field name="birthDate" column="birth_date" type="date" nullable="true"/>
        <field name="phoneExtension" column="phone_extension" type="string" length="10" nullable="true"/>

        <one-to-many field="roles" target-entity="App\Entity\UserRole" mapped-by="user" orphan-removal="true" fetch="EAGER">
            <cascade>
                <cascade-persist/>
                <cascade-remove/>
            </cascade>
        </one-to-many>

    </entity>

</doctrine-mapping>
```
After db column name has changed the login form does not show - I can see a doctrine error, as expected.
I tried to add `<id name="id" column="id_users" type="integer"/>` to the doctrine mapping - and then login form shows, but after submit I can see the following error: `Property "id" in "App\Entity\User" was already declared, but it must be declared only once`.

Is there any possibility to change the name of id column?

Thanks much once again.

**Comment by ro0NL** [comment]
See https://github.com/msgphp/msgphp/issues/87 :)

will be fixed in 0.2

**Comment by crabnky** [comment]
OK, I see, thanks. When do you expect version 0.2 will be released?

**Comment by ro0NL** [comment]
aiming for next week :)

**Comment by crabnky** [comment]
OK, thanks much. So will add some temprorary solution and wait :)

**Comment by ro0NL** [comment]
See v0.2.0
